lib/url.c

```diff
@@ -4711,14 +4711,16 @@ CURLcode Curl_init_do(struct Curl_easy *data, struct connectdata *conn)
 {
   struct SingleRequest *k = &data->req;
 
-  conn->bits.do_more = FALSE; /* by default there's no curl_do_more() to
-                                 use */
+  if (conn != NULL)
```

Comment:
we prefer these to be just `if(conn)` - and notice the lack of space before the paren!

Reply:
Updated as requested.